frontend/src/routes.tsx

```diff
@@ -59,7 +60,11 @@ const codePairRoutes: Array<CodePairRoute> = [
 	},
 	{
 		path: ":workspaceSlug",
-		element: <DocumentLayout />,
+		element: (
+			<PreviewRefProvider>
+				<DocumentLayout />
+			</PreviewRefProvider>
+		),
```

Comment:
I think it's better to move `PreviewRefProvider` into `DocumentLayout`.

Reply:
Removed in this [commit](https://github.com/yorkie-team/codepair/pull/238/commits/68fd0db5c8e37dcae8435051a3ee9dbfed2215aa).
Moved to backend implementation; ref removed as it's no longer needed in this [commit](https://github.com/yorkie-team/codepair/pull/238/commits/4b253e70c5b09314269946020125262cf85d8b64).